src/test/java/org/folio/rest/impl/AutomatedPatronBlocksAPITest.java

```diff
@@ -143,8 +143,14 @@ public void shouldReturnNoBlocksWhenUserSummaryDoesNotExist() {
 
   @Test
   public void shouldReturnNoBlocksWhenNoLimitsExistForPatronGroup() {
+    final String userId = randomId();
+    UserSummary userSummary = new UserSummary()
+      .withId(randomId())
+      .withUserId(userId);
+    waitFor(summaryRepository.save(userSummary));
+
```

Comment:
move to new test

Reply:
There is already a test for the case when UserSummary does not exist, so it is meaningless to add one more exact same test :
```
@Test
  public void shouldReturnNoBlocksWhenUserSummaryDoesNotExist() {
    String emptyBlocksResponse = toJson(new AutomatedPatronBlocks());
    sendRequestAndCheckResult(randomId(), emptyBlocksResponse);
  }
```